cmd/kindnetd/ethtool.go

```diff
@@ -64,10 +64,15 @@ func SetChecksumOffloading(ifName string, rxOn, txOn bool) error {
 	if txOn {
 		txVal = 1
 	}
-	_, err := ethtool(ifName, ethtoolSRxCsum, rxVal)
-	if err != nil {
+	// If rx checksum isn't supported ignore and continue.
+	if _, err := ethtool(ifName, ethtoolSRxCsum, rxVal); err != nil {
+		if sErr, ok := err.(syscall.Errno); ok && sErr == syscall.EAFNOSUPPORT {
+			return nil
```

Comment:
should we have to continue instead of returning? otherwise we never set the txCsum